core/src/test/java/org/testcontainers/junit/wait/strategy/AbstractWaitStrategyTest.java

```diff
@@ -118,4 +118,9 @@ protected void waitUntilReadyAndSucceed(GenericContainer container) {
         assertTrue(String.format("Expected container to be ready after timeout of %sms",
             WAIT_TIMEOUT_MILLIS), ready.get());
     }
+
+    protected String waitUntilReadyAndSucceedWithLogs(GenericContainer container) {
```

Comment:
I am not that happy about adding this method.

Reply:
If we proceed with checking logs, should I return the container itself?

Reply:
Method in general not necessary.